examples/sample_measurement/SampleMeasurement.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.SampleMeasurement_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
```

Comment:
Change these annotations to be about the actual sample measurement service.  The description should describe what the example actually does.  The collection should be "NI.Examples".  The tags should just be [].

Reply:
Yeah, I noticed this once I looked at the example US.  Will fix.

Reply:
Will revert this change since I made the correct fix in my PR of updating examples.